index.bs

```diff
@@ -10377,16 +10377,16 @@ a lower bit-depth), and by quantizing in time resolution
 {{AudioWorkletProcessor}}.
 
 <pre line-numbers class="example" highlight="js" title="BitCrusher - Global Scope">
-let context = new AudioContext();
+const context = new AudioContext();
 context.audioWorklet.addModule('bitcrusher.js').then(() =&gt; {
-	let osc = new OscillatorNode(context);
-	let amp = new GainNode(context);
+	const osc = new OscillatorNode(context);
```

Comment:
It is a little problem that constructor and factory method of `AudioNode` mix in sample code. I think that it is better to unify constructor or factory method than this.

Reply:
We should use the constructor where it is possible.